yarn.lock

```diff
@@ -2027,6 +2027,13 @@ axios@0.22.0:
   dependencies:
     follow-redirects "^1.14.4"
 
+axios@^0.24.0:
```

Comment:
do we need axios as well as the custom `api/apiCall` you added already? nest recommends using their HttpService which wraps axios https://docs.nestjs.com/techniques/http-module